test/src/Development/IDE/Test.hs

```diff
@@ -76,7 +83,11 @@ expectNoMoreDiagnostics timeout = do
     ignoreOthers = void anyMessage >> handleMessages
 
 expectDiagnostics :: [(FilePath, [(DiagnosticSeverity, Cursor, T.Text)])] -> Session ()
-expectDiagnostics expected = do
+expectDiagnostics
+  = expectDiagnostics_ . map (\(fp, dd) -> (fp, map (\(ds, c, t) -> (ds, c, t, Nothing)) dd))
+
+expectDiagnostics_ :: [(FilePath, [(DiagnosticSeverity, Cursor, T.Text, Maybe DiagnosticTag)])] -> Session ()
+expectDiagnostics_ expected = do
```

Comment:
Ususally `foo_` would be the version of `foo` that does slightly less than `foo`, e.g. doesn't return some info. Could we call `expectDiagnostics_` something like `expectedDiagnosticsTags`?

Reply:
I've renamed it to `expectDiagnosticsWithTags`.